index.html

```diff
@@ -76,6 +76,11 @@ <h2>Conferences for the JavaScript community.</h2>
           <p>JSConf China</p>
         </div>
 
+        <div class="conf">
```

Comment:
Please move this into second position. We roughly sort this by immediate future relevance.